stetho/src/main/java/com/facebook/stetho/inspector/elements/android/window/WindowWrapper.java

```diff
@@ -0,0 +1,15 @@
+package com.facebook.stetho.inspector.elements.android.window;
+
+import android.view.View;
+
+public class WindowWrapper {
```

Comment:
nit: I think you also mean `WindowDecorViewWrapper` or maybe even better, `WindowDecorViewHolder`